src/components/.storybook/config.js

```diff
@@ -0,0 +1,15 @@
+import { configure } from '@storybook/react'
+import { setOptions } from '@storybook/addon-options'
+
+setOptions({
+  name: 'Whiskey Notes'
+})
+
+// import all component stories
+const req = require.context('../', true, /\.story\.js$/)
```

Comment:
configuration file for Storybook. basically just gets all files ending in `.story.js` and loads them